packages/react/src/components/footer/footerBase/FooterBase.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+// import core base styles
+import 'hds-core';
+import styles from './FooterBase.module.scss';
+
+export type FooterBaseProps = React.PropsWithChildren<{
+  /**
+   * Text to be displayed next to the copyright symbol
+   */
+  copyrightHolder?: React.ReactNode;
+  /**
+   * Text to be displayed after the copyright holder text
+   */
+  copyrightText?: React.ReactNode;
+}>;
+
+export const FooterBase = ({ children, copyrightHolder, copyrightText }: FooterBaseProps) => {
+  const year = new Date().getFullYear();
+
+  return (
+    <section className={styles.base}>
+      <hr className={styles.divider} />
+      <div className={styles.copyright}>
+        <span className={styles.copyrightHolder}>
+          © {copyrightHolder} {year}
```

Comment:
What will this render if `copyrightHolder` is undefined?

Reply:
© 2020